test_project/settings.py

```diff
@@ -193,6 +193,7 @@
     'CONVERSION_SERVER': 'http://localhost/',
     'CAPTURE_SERVER': 'http://localhost/',
     'SENDFILE_HTTP_HEADER': 'X-Accel-Redirect',
+    'MAX_CHARACTERS': 1200,
```

Comment:
let MAX_CHARACTERS to None and test with @override_settings, with and without value

Reply:
J'ai réussi à faire quelque chose, et je n'utilise plus `app_settings` dans le formulaire.